### ISSUE TITLE
Add support for .new_lb files

### ISSUE THREAD

**Issue by zentol** [Synthlight/MHW-Editor#2]
Defines some costs for MR weapon augmentations (no custom upgrades).

10b offset: 6b prefix, int32 array_size

```
uint32 	aug_category (0=aug_slots, 1=atk, 2=aff, 3=def, 4=slot, 5=health, 6=element)
uint32 	aug_level
uint32 	aug_slot_cost
uint32	research_cost (R10)
uint32	research_cost (R11)
uint32	research_cost (R12)
```

Modifying research_cost only seems to work with multiples of 100.

**Comment by Synthlight** [comment]
Not just a request, but struct info too. Thanks!

I'll probably add both this and the other (eq_crt) and release in a day or so.

**Comment by Synthlight** [comment]
Here's a few screenshots of the file assuming I got everything right.
Poke me if anything looks off.
![newLb](https://user-images.githubusercontent.com/1075166/73603901-ed5cb000-4556-11ea-86d7-1578bdf31262.png)

**Comment by Synthlight** [comment]
Done and released in v1.6.
